agent/eni/networkutils/utils_linux_test.go

```diff
@@ -95,11 +95,11 @@ func TestGetMACAddressNotFoundRetry(t *testing.T) {
 		}, nil),
 	)
 	ctx := context.TODO()
-	// Set max retry duration to twice that of the min backoff to ensure that there's
+	// Set max retry duration to twice that of the max backoff to ensure that there's
 	// enough time to retry
-	mac, err := GetMACAddress(ctx, 2*macAddressBackoffMin, randomDevice, mockNetlink)
+	mac, err := GetMACAddress(ctx, 2*macAddressBackoffMax, randomDevice, mockNetlink)
```

Comment:
can it be just `macAddressBackoffMax` instead of `2* macAddressBackoffMax`?

Reply:
Probably not, the test is expecting two calls(retries), so in the worst case if the first retry has a backoff `macAddressBackoffMax`, then the second call wouldn't happen due to the timeout.

Reply:
correct me if i'm wrong - but we were thinking that since there are two calls, and the [backoff](https://github.com/aws/amazon-ecs-agent/blob/master/agent/utils/backoff.go#L42) could actually be `max + max*jitterMultiple`, which would put us past the `macAddressBackoffMax` upper bound for the retry call to be invoked.